Userland/Libraries/LibCodeComprehension/Types.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2022, Itamar S. <itamar8910@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/String.h>
+
+#pragma once
+
+namespace CodeComprehension {
+
+enum class Language {
+    Unspecified,
+    Cpp,
+};
```

Comment:
Why is there no Shell here?

Reply:
Turns out we leave `AutocompleteResultEntry::language` as `Unspecified` in many places, and yeah we don't use it at all for Shell stuff.

It doesn't look like something that should be fixed as part of this PR though, so I added a TODO for that :)